.github/ISSUE_TEMPLATE/bug-en.md

```diff
@@ -0,0 +1,91 @@
+---
+name: Bug Report
+about: Create a bug report of Xray.
+title: '[Bug] <bug you are reporting>'
+labels: ''
+assignees: ''
+
+---
+
+<!-- Thanks for your reporting.
+1. Please make sure you are submitting a bug of Xray-core. If you are not sure, please give feedback in our telegram group.
```

Comment:
```suggestion
1. Please make sure you are submitting a bug of Xray-core rather than a bug of clients such as Qv2ray, v2rayN, v2rayNG, luci-app-ssr-plus, luci-app-passwall. If you are not sure, please give feedback in our telegram group.
```